install.html

```diff
@@ -257,67 +257,57 @@ <h4>7. Debug mode</h4>
-                    Authorize an application for github:<br />
-                    <a href="https://github.com/settings/developers">https://github.com/settings/developers</a>
-                </p>
-            </li>
-            <li><b>OAUTH2 over IMAP</b>
-                <p>Gmail and Outlook.com support OAUTH2 authentication over IMAP. This is preferable to normal IMAP
-                    authentication because Cypht never has access to your account password.<br /><br />
-                    Example oauth2 ini file:<br />
-                    <a
-                        href="https://github.com/cypht-org/cypht/blob/master/modules/imap/oauth2.ini">https://github.com/cypht-org/cypht/blob/master/modules/imap/oauth2.ini</a><br /><br />
-                    Authorize an application for gmail<br />
-                    <a
-                        href="https://console.developers.google.com/project">https://console.developers.google.com/project</a><br /><br />
-                    Authorize an application for outlook.com<br />
-                    <a
-                        href="https://account.live.com/developers/applications/">https://account.live.com/developers/applications/</a><br /><br />
-                </p>
-            </li>
-            <li><b>LDAP contacts</b>
-                <p>Cypht can use an LDAP server for contacts.<br /><br />
-                    Example ldap.ini file:<br />
-                    <a
-                        href="https://github.com/cypht-org/cypht/blob/master/modules/ldap_contacts/ldap.ini">https://github.com/cypht-org/cypht/blob/master/modules/ldap_contacts/ldap.ini</a><br /><br />
-                </p>
-            </li>
-            <li><b>WordPress</b>
-                <p>Cypht can aggregate WordPress.com notifications.<br /><br />
-                    Example wordpress.ini file:<br />
-                    <a
-                        href="https://github.com/cypht-org/cypht/blob/master/modules/wordpress/wordpress.ini">https://github.com/cypht-org/cypht/blob/master/modules/wordpress/wordpress.ini</a><br /><br />
-                    Authorize an application for WordPress.com:<br />
-                    <a href="https://developer.wordpress.com/apps/">https://developer.wordpress.com/apps/</a>
-                </p>
-            </li>
-            <li><b>Custom themes</b>
-                <p>You can create your own themes for Cypht. Edit the themes.ini file to include your theme, and put the
-                    css
-                    file in modules/themes/assets.</p><br />
-                <p>
-                    <a
-                        href="https://github.com/cypht-org/cypht/blob/master/modules/themes/themes.ini">https://github.com/cypht-org/cypht/blob/master/modules/themes/themes.ini</a>
-                </p>
-            </li>
-
+          <li><b>Github</b>
```

Comment:
Oh, do you mean the indentation between the <ul> and <li><b>Github</b>? If so, that is the indentation that was used in install-2.x.html, which uses 2 spaces instead of 4! Do I switch it back to 4 for install.html? Or did you mean a different part of the code?

Reply:
I get you. I am am reviewing based on git changes... As you can see it's like they are yours yet not 😊.

Reply:
No problem! Ill change the p tag closing in a bit, do I change anything on this part?